frontend/src/config/jordan/layers.json

```diff
@@ -4015,10 +4015,16 @@
     ],
     "legend_text": "Groundwater developed for irrigation (MCM) "
   },
-  "cdi_v1": {
-    "title": "Combined Drought Index (v1)",
+  "cdi_v1_monthly": {
```

Comment:
Note: I chose to use different layers for monthly vs seasonal. This was primarily driven by the need for different options in the layers' "available dates" for time selection and querying